src/pipeline.cc

```diff
@@ -1209,6 +1218,11 @@ NAN_METHOD(pipeline) {
   baton->pngCompressionLevel = AttrTo<uint32_t>(options, "pngCompressionLevel");
   baton->pngAdaptiveFiltering = AttrTo<bool>(options, "pngAdaptiveFiltering");
   baton->webpQuality = AttrTo<uint32_t>(options, "webpQuality");
+  baton->webpAlphaQuality = AttrTo<uint32_t>(options, "webpAlphaQuality");
+  baton->webpLossless = AttrTo<bool>(options, "webpLossless");
+  if(HasAttr(options, "webpNearLossless")) {
```

Comment:
Does this attribute need the `HasAttr` check given there is (or should be) a default?